src/components/dashboard/main-content/course-enrollments/data/utils.js

```diff
@@ -1,6 +1,26 @@
 import moment from 'moment';
 import { COURSE_STATUSES } from './constants';
 
+/**
+ * Determines whether a course enrollment may be unenrolled based on its enrollment
+ * status (e.g., in progress, completed) and enrollment completion.
+ *
+ * @param {Object} courseEnrollment Data for an enterprise course enrollment.
+ * @returns True if the enrollment can be unenrolled. False if not.
+ */
+export const canUnenrollCourseEnrollment = (courseEnrollment) => {
+  const unenrollableCourseRunTypes = [
+    COURSE_STATUSES.inProgress,
+    COURSE_STATUSES.upcoming,
+    COURSE_STATUSES.completed,
+    COURSE_STATUSES.savedForLater,
+  ];
+  return (
+    unenrollableCourseRunTypes.includes(courseEnrollment.courseRunStatus)
```

Comment:
nit: there are only 4 items here so it's probably not a big deal, but would set().has be a better choice if we're just looking up if an item is exists?